about.html

```diff
@@ -46,8 +46,8 @@
             <div id="mobilemenu">
                 <div id="option">
                     <div class="login">Login</div>
-                    <a class="link" href="index.html">Home</a>
-                    <a class="link" href="about.html">about</a>
+                    <a class="link" href="about.html">Home</a>
+                    <a class="link" href="index.html">about</a>
```

Comment:
Great job working on this issue as requested by the previous reviewer.
The change requested by the first reviewer doesn't concern these lines so please reverse back them:

- [ ] Please make sure the `home` ([line 49-50](https://github.com/Gardimy/Gardimy-project/blob/capstone2/about.html#L49-L50)) page redirects to the home page (index.html) and the `about` redirects to the about page (about.html)

The first reviewer was talking about [line 27](https://github.com/Gardimy/Gardimy-project/blob/capstone2/about.html#L27-L30).

- [ ] Please make sure the `home` [line 27](https://github.com/Gardimy/Gardimy-project/blob/capstone2/about.html#L27-L30) page redirects to the home page (index.html) and the `about` ([line 30](https://github.com/Gardimy/Gardimy-project/blob/capstone2/about.html#L30)) redirects to the about page (about.html).

This will make your project way more accessible which will improve the user experience 📱  😄 👍

Reply:
I already fix it.! please check the last commit.